.requirement-checker/src/Checker.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace HumbugBox383\KevinGH\RequirementChecker;
```

Comment:
we can avoid that dump for now, I'll do one manually for the next release

Reply:
reverted